torch/_dynamo/guards.py

```diff
@@ -2203,6 +2212,9 @@ def cleanup_builder(weak_b):
         # Break retain cycle. See test_release_input_memory
         w_builder = weakref.ref(builder, cleanup_builder)
 
+        if guard_filter_fn:
+            guards = guard_filter_fn(guards)
```

Comment:
I understand `guard_filter_fn` is used here. But how about `symbolic_guard_filter_fn`? where is it used?

And, if I use `guard_filter_fn`, will it be applied to symbolic guards as well?

Reply:
No, that's the clunky part. The symbolic guards come from a different place. And therefore we need 2 different hooks.